R/reporter-junit.R

```diff
@@ -26,17 +26,18 @@ classnameOK <- function(text) {
 #' \url{http://llg.cubic.org/docs/junit/}
 #'
 #' @export
-JunitReporter <- R6::R6Class("JunitReporter", inherit = Reporter,
+JunitReporter <- R6::R6Class(
+  "JunitReporter", inherit = Reporter,
   public = list(
-    results  = NULL,
-    timer    = NULL,
-    doc      = NULL,
-    errors   = NULL,
+    results = NULL,
```

Comment:
I wonder if there's a way that styler could detect if the `=` were aligned, and if so, leave as is